backends/jwt_test.go

```diff
@@ -1654,3 +1654,77 @@ func TestJWTFormTextResponseServer(t *testing.T) {
 	})
 
 }
+
+func TestJWTHttpDelayedResponse(t *testing.T) {
+	topic := "test/topic"
+	var acc = int64(1)
+	clientID := "test_client"
+	token, _ := jwtToken.SignedString([]byte(jwtSecret))
+	// wrongToken, _ := wrongJwtToken.SignedString([]byte(jwtSecret))
+
+	version := "2.0.0"
+
+	rightToken := token
+
+
+	mockServer := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		time.Sleep(5 * time.Second)
```

Comment:
To avoid wasting time in tests, you could do the following tests:
- Set the `sleep` to just a bit over a second, e.g. `time.Sleep(1200 * time.Millisecond)`, initialize a JWT backend with a 1 second timeout and expect the check to fail due to a timeout.
- Set no `sleep` and initialize a JWT backend with a 1 second timeout and see the check succeed.

Since under the hood all checks use the same client to do the request, you can do one in e.g. user check and the other in the acl, that is move the sleep inside one of the cases below and leave the other one with no sleep.
That way you can both reuse the mock server instead of needing a second one, and you don't need to initialize your JWT backend twice since the same 1 second timeout will work to fail a long check and succeed on a quick one.